src/types/Theme.ts

```diff
@@ -97,9 +104,15 @@ export interface Theme {
   loadingIcon: string
 
   // Background
-  backgroundGradientColors: string[]
-  backgroundGradientStart: { x: number; y: number }
-  backgroundGradientEnd: { x: number; y: number }
+  background: {
+    blurR: number
+    color: string
+    dotOpacity: number
+    dots: ThemeDot[]
+  }
```

Comment:
I like the extra level for the theme file, should we be doing this for things like inputs and other components?